samples/quickstart.js

```diff
@@ -1,38 +1,36 @@
-// Copyright 2020 Google LLC
-//
+// Copyright 2020, Google LLC.
 // Licensed under the Apache License, Version 2.0 (the "License");
 // you may not use this file except in compliance with the License.
 // You may obtain a copy of the License at
 //
-//     https://www.apache.org/licenses/LICENSE-2.0
+//    http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
this is a little weird, probably just leave it as `https://`, must be something we updated in our license at some point.

Reply:
Updated.